src/panels/config/entities/editor-tabs/settings/entity-settings-helper-tab.ts

```diff
@@ -120,32 +117,30 @@ export class EntityRegistrySettingsHelper extends LitElement {
     }
     const stateObj = this.hass.states[this.entry.entity_id];
     return html`
-      <paper-dialog-scrollable .dialogElement=${this.dialogElement}>
-        ${this._error ? html` <div class="error">${this._error}</div> ` : ""}
-        <div class="form">
-          ${!this._componentLoaded
-            ? this.hass.localize(
-                "ui.dialogs.helper_settings.platform_not_loaded",
-                "platform",
-                this.entry.platform
-              )
-            : this._item === null
-            ? this.hass.localize("ui.dialogs.helper_settings.yaml_not_editable")
-            : html`
-                <div @value-changed=${this._valueChanged}>
-                  ${dynamicElement(`ha-${this.entry.platform}-form`, {
-                    hass: this.hass,
-                    item: this._item,
-                    entry: this.entry,
-                  })}
-                </div>
-              `}
-          <ha-registry-basic-editor
-            .hass=${this.hass}
-            .entry=${this.entry}
-          ></ha-registry-basic-editor>
-        </div>
-      </paper-dialog-scrollable>
+      ${this._error ? html` <div class="error">${this._error}</div> ` : ""}
+      <div class="form">
+        ${!this._componentLoaded
+          ? this.hass.localize(
+              "ui.dialogs.helper_settings.platform_not_loaded",
+              "platform",
+              this.entry.platform
+            )
+          : this._item === null
+          ? this.hass.localize("ui.dialogs.helper_settings.yaml_not_editable")
+          : html`
+              <div @value-changed=${this._valueChanged}>
```

Comment:
If we just care about capturing events, let's use a span so we don't accidentally do layout stuff.